.gitignore

```diff
@@ -14,7 +14,6 @@ artifacts/
 **/.vscode/
 
 cli/
-cli1.0.4/
```

Comment:
I wouldn't remove this just yet.
This was useful in earlier versions, and there's no harm in keeping it.

Some of us have to switch to older versions of code sometimes and it would mess with our builds.

Reply:
You have git for that, right?

Reply:
The problem is that if I checkout an old branch and run `configure.ps1`, it will create the `cli1.0.4` folder, and be correctly ignored. Now, if I checkout dev, I still have `cli1.0.4`, but if `.gitignore` is no longer ignoring `cli1.0.4`, then it's going to want to track those files.

Reply:
Ahhh... I see now! 🤯